### ISSUE TITLE
License for Version 2 of library - Request for Clarification

### ISSUE THREAD

**Issue by ReallyNiceGuy** [jpivarski/svgfig#18]
Hello, according to issue #4, you prefer to keep the license for version 1 as GPL, but could you please clarify which license is used for version 2?
As I understood from the issue referred above, you would use a less restrictive one. Would you be so kind to add the license file or the license header into the source code?
I would like to use it in a commercial product, if you would not object.

Thank you in advance

**Comment by jpivarski** [comment]
Hi again! I made a new release, this time with a BSD 3-clause license. (It's what I use on all of my projects nowadays, and I know that it will work for everybody.)

I found our old conversation (#17), when I was apparently at my previous job. Now even that—Augustus—is obsolete! I think there ought to be a way to do SVG-wrapping in Python as a library, rather than this plotting/user interface-oriented tool, but if you really want to use this, I have no objections.

Check this out: https://web.archive.org/web/20081110144209/https://code.google.com/p/svgfig/

Now 14 years old!

![image](https://user-images.githubusercontent.com/1852447/154134806-09a9440f-25f8-4a99-bf57-a4cf2aaf60b9.png)
